### ISSUE TITLE
Bug in Run-AlPipeline code

### ISSUE THREAD

**Issue by ernestasjuska** [microsoft/navcontainerhelper#1369]
**Describe the issue**

ValidateSet attribute does not make sense in its current place.

```powershell
    [switch] $assignPremiumPlan,
    [Parameter(Mandatory=$false)]
    [ValidateSet('None','All','Essential','Premium')]
    [string] $tenant = "default",
    [string] $memoryLimit,
    [string] $memoryLimit,
```

It was introduced in this commit: https://github.com/microsoft/navcontainerhelper/commit/eb475a05a5aeb029ce6f619e8da072b254720d22

**Comment by freddydk** [comment]
Correct, that is a bug.
You can remove the lines in your downloaded file if need be, next preview, they are gone.

Thanks

**Comment by freddydk** [comment]
Fix shipped in 1.0.10
